fhir-aggregator-scripts/bulk-import-request-TCGA-KIRC.json

```diff
@@ -0,0 +1,204 @@
+{
+  "resourceType": "Parameters",
+  "parameter": [
+    {
+      "name": "inputFormat",
+      "valueCode": "application/fhir+ndjson"
+    },
+    {
+      "name": "inputSource",
+      "valueUri": "https://fhir-aggregator-public.s3.us-west-2.amazonaws.com"
```

Comment:
Would love to have access to S3 buckets on my end (if $ allows) to validate my ETLs.